scraping/scraping/spiders/zalando.py

```diff
@@ -62,9 +62,8 @@ def parse_SERP(self, response: SplashJsonResponse) -> Iterator[SplashRequest]:
             )
 
         # Pagination: Parse next SERP 'recursively'
-        next_page = response.css('[title="nächste Seite"]::attr(href)').extract_first()
-
-        if next_page:
+        for next_page in response.css('[class="DJxzzA PgtkyN"]::attr(href)').getall():
```

Comment:
This should produce an endless loop? Since you always scrape the page before and after.

Reply:
no. scrapy filters duplicate request automatically.